### DISCUSSION TITLE
What versions of Rhino does BHoM 7.0.B.0 installer support?

### DISCUSSION THREAD

**Discussion by mtsmith** [BHoM/BHoM#1599] [Q&A]
BHoM shows up in Excel and Revit 2024 but not Rhino 8. Is compatibility listed anywhere in the docs?

**Comment by FraserGreenroyd** [comment]
Hi @mtsmith we currently support Rhino 6 and 7, however we have not yet had a chance to check compatibility with Rhino 8 and make any changes which might be required to support it.

It is on the cards to be looked at this year as we're aware Rhino 8 is going to be the newer replacement that everyone starts adopting.

There are supported versions listed in the readmes for both [Rhino_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit?tab=readme-ov-file#known-versions-of-software-supported) and [Grasshopper_Toolkit](https://github.com/BHoM/Grasshopper_UI?tab=readme-ov-file#known-versions-of-software-supported), however, it would appear both of these are out of date as they mention Rhino 5 but not Rhino 7 😱 

We are doing a big push in the community this year around [documentation](https://bhom.xyz/documentation) to try and keep on top of what our community can offer the AEC industry so hopefully it will be listed properly at some point - but feel free to take it up if one of the maintainers doesn't get to it first 😄
